db/listener.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app");
```

Comment:
We don't need a listener file as we're only supertest which has this functionality inbuilt

Reply:
riiiight i realised that but should have removed it.